### ISSUE TITLE
Closed issues from github aren't removed from sidebar

### ISSUE THREAD

**Issue by kgn** [Me1000/issues#130]
I pushed some commits that closed some issues but I still see them in the open issues list, they aren't removed even though they are closed on github.com.

**Comment by Me1000** [comment]
Can you confirm they're removed when you pull to reload?

**Comment by kgn** [comment]
Yes it is still there even after a refresh, unfortunately this is in a private repo. This probably won't help but here is a video of what I'm seeing: http://cl.ly/1D3j0e1T0C071Y1i190Z Any chance this could be related to #132?

**Comment by Me1000** [comment]
Nah, I think it's just a bug. I believe if you select the issue and hit the reload button in the detail view, it will be removed. 

This _is_ a bug though. Thanks for letting me know about it.

**Comment by kgn** [comment]
Yep reloading the issue did the trick
